### DISCUSSION TITLE
'operation not permitted' when running installs

### DISCUSSION THREAD

**Discussion by ambethell** [Homebrew/discussions#4477] [Everyday usage]
### Output of `brew config`

```shell
bash-3.2$ brew config
HOMEBREW_VERSION: 4.0.16
ORIGIN: https://github.com/Homebrew/brew
HEAD: 01e0c20d01c6aab28d844280d130f0e745e2ddd9
Last commit: 3 days ago
Core tap JSON: 03 May 10:15 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_blizzard_avalanche
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 13.3.1-arm64
CLT: 14.3.0.0.1.1679647830
Xcode: N/A
```

### Output of `brew doctor`

```shell
bash-3.2$ brew doctor
Your system is ready to brew.
```

### Description of issue

Certain apps refuse to install (although I'm certain they did before) - such as 'wget' or 'mkcert'.

I've uninstalled and re-installed brew multiple times - there are no errors on the install script only when attempting to install certain apps.  I've also uninstalled and re-installed xcode - but it always comes up as N/A on brew config.

I get the following error when attempting to install 'wget'.

> cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgconv: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgconv: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msggrep: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msggrep: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgcomm: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgcomm: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgen: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgen: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgmerge: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgmerge: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/envsubst: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/envsubst: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msginit: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msginit: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/gettext: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/gettext: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgcmp: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgcmp: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgcat: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgcat: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/recode-sr-latin: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/recode-sr-latin: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgexec: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgexec: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/ngettext: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/ngettext: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgunfmt: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgunfmt: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgfilter: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgfilter: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgfmt: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgfmt: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msgattrib: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msgattrib: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/xgettext: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/xgettext: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/bin/msguniq: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/bin/msguniq: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/lib/gettext/urlget: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/lib/gettext/urlget: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/lib/gettext/hostname: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/lib/gettext/hostname: Operation not permitted
cp: /private/tmp/d20230503-52805-19bnr5a/gettext/./0.21.1/lib/gettext/cldr-plurals: could not copy extended attributes to /opt/homebrew/Cellar/gettext/0.21.1/lib/gettext/cldr-plurals: Operation not permitted
Error: Failure while executing; `/usr/bin/env cp -pR /private/tmp/d20230503-52805-19bnr5a/gettext/. /opt/homebrew/Cellar/gettext` exited with 1. Here's the output:

It's not clear what I need to run to fix this.  Any ideas?  I'm on a m2 macbook - macOS ventura 13.3.1 (a) - this computer also has Cortex XDR installed.  I have given terminal / iterm2 full disk access in privacy & security.

**Comment by mgastel** [comment]
I have the exact same issue.  Brew worked a few weeks ago but now fails.  I'm running 13.5.2 on M2.   I've run /bin/cp and cp found in coreutils manually (I assume brew is using one of the two), and in both cases it works and the extended attribute was copied.   I'm thinking this is Cortex XDR / CyberArk EPM but not sure how to isolate the problem, or potentially get more information that I can pass on.

(my personal laptop works fine, so the corporate ware is the prime suspect)
